src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
Hm, looks like moving all annotations to `org.scalameter.japi` can cause conflicts (using wildcard import) with old java frontend. For example, we have both `Warmer` class in `org.scalameter` and `org.scalameter.japi`. 
The first can be used in the new Java API, whilst the second is only used in the old Java frontend.
So maybe these annotations should be moved to `org.scalameter.japi.annotation`?